OSDOCS-18201: Added note about default NPs OCP includes in its namesp… (#110722)
openshift/openshift-docs: modules/nw-networkpolicy-about.adoc

By default, all pods in a project are accessible from other pods and network endpoints. To isolate one or more pods in a project, you can create `NetworkPolicy` objects in that project to indicate the allowed incoming connections. Project administrators can create and delete `NetworkPolicy` objects within their own project.

[IMPORTANT]
====

> From {product-title} 4.22, {product-title} now includes `NetworkPolicy` objects in its own namespaces by default. This inclusion improves overall security and better protects control plane components. Do not modify the `NetworkPolicy` objects that {product-title} includes in its own namespaces by default.

Reviewer: ```suggestion
From {product-title} 4.22, {product-title} now includes `NetworkPolicy` objects in some of its own namespaces by default. This inclusion improves overall security and better protects control plane components. Do not modify the `NetworkPolicy` objects that {product-title} includes in its own namespaces by default.
```

And this is an ongoing process.

By default, all pods in a project are accessible from other pods and network endpoints. To isolate one or more pods in a project, you can create `NetworkPolicy` objects in that project to indicate the allowed incoming connections. Project administrators can create and delete `NetworkPolicy` objects within their own project.

[IMPORTANT]
====
From {product-title} 4.22, {product-title} now includes `NetworkPolicy` objects in some of its own namespaces by default. This inclusion improves overall security and better protects control plane components. Do not modify the `NetworkPolicy` objects that {product-title} includes in its own namespaces by default. To check the namespaces that include the objects by default, you can run the following command:

[source,terminal]
----
$ oc get networkpolicies --all-namespaces
----

> The {product-title} {product-version} release did not include these objects in all {product-title} namespaces; later {product-title} releases will include the objects in these namespaces. 

Reviewer 2: I'm a little confused by this product-version attribute. The sentence is worded in the past, but right now this would also say 4.22. Should it be present tense, since it will always resolve the current release?

Also, the second part of the sentence is predicting the future, which we should avoid. I'd recommend making this less certain, in case plans change.

Suggestion for both below:

```suggestion
The {product-title} {product-version} release does not include these objects in all {product-title} namespaces; later {product-title} releases might include the objects in additional namespaces. 
```
